.golangci.yaml

```
@@ -10,6 +10,24 @@ linters-settings:
     disabled-checks:
       - appendAssign
       - exitAfterDefer
+  goheader:
+    values:
+      regexp:
+        CopyrightDate: "Copyright \\d{4}"
+    template: |-
+      {{CopyrightDate}} The Kubernetes Authors.
```

```suggestion
     The Kubernetes Authors.
```

The current template without year is intended.
We drop the year since we want to avoid the license update every year: https://github.com/kubernetes-sigs/kueue/pull/1590

---

Additionally, could we reuse this template instead of hardcoded one here?

https://github.com/kubernetes-sigs/kueue/blob/main/hack/boilerplate.go.txt

---

I investigated PR #1590 before creating this one, and what I understood from the description is that it is only for generated code. This PR touches only human-added files, not the generated files.

In human-added files, there is no issue with updating the year every year. We add the year into license only on introducing a new file.

Are you sure I need to change the copyright template to be the same for human-added and generated files? It requires me to update roughly 6K files. It's not a big deal, just asking to be sure.

---

Removed year from all files.

`template-path` can't be used, please see the explanation comment in the `.golangci.yaml`.